ansible/roles/dev/files/django.bashrc

```diff
@@ -1 +1,3 @@
 export DJANGO_SETTINGS_MODULE=pulpcore.app.settings
+export PULP_DEBUG=True
+export PULP_SECRET_KEY=$(cat /dev/urandom | tr -dc 'a-z0-9!@#$%^&*(\-_=+)' | head -c 50)
```

Comment:
wouldn't this change every time you logged in? Also mind the missing new line ;)